public/statement.css

```diff
@@ -36,7 +36,7 @@
     }
 
     .logo img {
-        width: 2cm;
+        height: 2cm;
```

Comment:
Actually, I think that the discussion was to limit support to square logos. In order to be robust, I would rather express the size of the image in terms of relative width/height and limit the other dimension to prevent any overflow.

Reply:
After a few tries, this solution seems to constrains the logo in a 6x2 cm rectangle while keeping height/width ratio:
```css
.logo img {
    height: auto;
    max-height: 2cm;
    width: auto;
    max-width: 6cm;
}
```